stable/nginx-ingress/values.yaml

```diff
@@ -354,6 +354,11 @@ defaultBackend:
 rbac:
   create: true
 
+# If true, create & use Pod Security Policy resources
+# https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+podSecurityPolicy:
+  enabled: false
```

Comment:
would it be good to configure the ci-tests to run with this enabled to get test-coverage?
See https://github.com/helm/charts/tree/master/test

Reply:
Already committed. Let's see what gives.

Reply:
seems it worked just fine!